backend/src/flotilla/api/missions_api.py

```diff
@@ -70,12 +74,14 @@ async def read_single_mission(
     try:
         echo_response: RequestResponse = echo_requests.get_mission(mission_id)
     except RequestException:
-        response.status_code = 404
-        return ProblemDetails(title="Not found - Could not contact echo", status=404)
+        response.status_code = 502
```

Comment:
Instead of writing status codes by writing the int directly an enum should be used. This goes for both in code and documentation decorators.

Use this library: https://docs.python.org/3/library/http.html
![image](https://user-images.githubusercontent.com/14331473/152939031-b90cfb66-bedc-48c7-bcbe-b6a71281008b.png)